scripts/variantstore/wdl/ngs_filter_extract.wdl

```diff
@@ -0,0 +1,497 @@
+        max_location=$(echo "( ~{chrom_id} + 1 ) * ~{chrom_offset}" | bc)
+
+        gatk --java-options "-Xmx4g" \
+            ExtractFeatures \
+                --ref-version 38  \
+                -R "~{reference}" \
+                -O "~{output_file}" \
+                --local-sort-max-records-in-ram ~{local_sort_max_records_in_ram} \
+                --sample-table ~{fq_sample_table} \
+                --alt-allele-table ~{fq_alt_allele_table} \
+                --min-location ${min_location} --max-location ${max_location} \
+                --project-id ~{read_project_id}
+    >>>
+
+    # ------------------------------------------------
+    # Runtime settings:
+    runtime {
+        docker: "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_d8a72b825eab2d979c8877448c0ca948fd9b34c7_change_to_hwe"
+        memory: "7 GB"
+        disks: "local-disk 10 HDD"
+        bootDiskSizeGb: 15
+        preemptible: 3
+        cpu: 2
+    }
+
+    # ------------------------------------------------
+    # Outputs:
+    output {
+        File output_vcf = "~{output_file}"
+        File output_vcf_index = "~{output_file}.tbi"
+    }
+ }
+
+task UploadFilterSetToBQ {
+    # indicates that this task should NOT be call cached
+    
+    # TODO: should this be marked as volatile???
+    #meta {
+    #   volatile: true
+    #}
+
+    input {
+        # ------------------------------------------------
+        # Input args:
+
+        # Runtime Options:
+        File? gatk_override
+
+        String filter_set_name
+
+        File snp_recal_file
+        File snp_recal_file_index
+        File snp_recal_tranches
+
+        File indel_recal_file
+        File indel_recal_file_index
+        File indel_recal_tranches
+
+        String fq_info_destination_table
+        String fq_tranches_destination_table
+    }
+
+
+    # ------------------------------------------------
+    # Run our command:
+    command <<<
+        set -e
+        set -o pipefail
+
+        export GATK_LOCAL_JAR=~{default="/root/gatk.jar" gatk_override}
+
+        gatk --java-options "-Xmx4g" \
+            CreateFilteringFiles \
+            --ref-version 38 \
+            --filter-set-name ~{filter_set_name} \
+            -mode SNP \
+            -V ~{snp_recal_file} \
+            -O ~{filter_set_name}.snps.recal.tsv
+
+        gatk --java-options "-Xmx4g" \
+            CreateFilteringFiles \
+            --ref-version 38 \
+            --filter-set-name ~{filter_set_name} \
+            -mode INDEL \
+            -V ~{indel_recal_file} \
+            -O ~{filter_set_name}.indels.recal.tsv
+
+        # merge into a single file
+        echo "Merging SNP + INDELs"
+        cat ~{filter_set_name}.snps.recal.tsv ~{filter_set_name}.indels.recal.tsv | grep -v filter_set_name | grep -v "#"  > filter_set_load.tsv
+
+        # BQ load likes a : instead of a . after the project 
+        bq_info_table=$(echo ~{fq_info_destination_table} | sed s/\\./:/)
```

Comment:
that's weird that you need a : after the project. in our other scripts, we specify the project with `--project_id=~{project_id}`  and then we don't include it in the dataset.table. also, when i was doing a bq query, the : was needed when i was using legacy mode and the . otherwise.

Reply:
yeah you can supply the default project with --project_id OR can you supply a fully qualified name.  Since for all the other scripts we tend to take in a FQ table name (project.dataset.table) I kept this the same from an input perspective for user consistency, but that means we have to fiddle it here to get it in the right format for `bq load`